doc/install/build_install.rst

```diff
@@ -456,39 +455,44 @@ use ``ARB_PYTHON_LIB_PATH`` to specify the location where the Python module is t
 
     cmake .. -DARB_WITH_PYTHON=on -DARB_PYTHON_PATH_LIB=/custom/path
 
-If CMake is run in a `venv` or Conda environment, set ``CMAKE_INSTALL_PREFIX`` to the
-base path of the venv. The example below shows a workflow that creates a virtual
-environment, then installs Arbor inside the environment.
+.. note::
+    The location of libraries under a prefix in only guaranteed to be standard for Python's global library location.
+    Therefore, correct installation of the Python package to any other location using ``CMAKE_INSTALL_PREFIX``,
+    such as user directory (e.g. `~/.local`), a Python or Conda virtual environment, may result in installation to a wrong path.
```

Comment:
So the issue is that under the prefix, the `/lib[arch]` dir is up to the implementation/Python distro. We don't know whether it will be `/lib` or `/lib64`, hence the warning in the Python docs. The only way to be sure of the path, is to not assume a directory layout under the prefix and ask Python for the full path, for instance with `python3 -m site --user-site`, and set `ARB_PYTHON_LIB_PATH`. (I did not know this, until I ran into such a Python distro.)

So, I think it's important to advise the user that setting a prefix is not guaranteed to work for installs except in `/usr`, where Python _does_ promise the directory layout we all know and love. The only way to be sure it's installed in a path that is used, is to 1. use pip, not CMake; or 2. prod at Python yourself to find the particular path it uses and set it with `ARB_PYTHON_LIB_PATH`. The issue with 2. is that there is no single function to obtain this path (also depends on whether you want a user, venv, system install).